doc/modules/decomposition.rst

```diff
@@ -864,16 +864,47 @@ The graphical model of LDA is a three-level Bayesian model:
 .. image:: ../images/lda_model_graph.png
    :align: center
 
+
+Note on notations presented in the graphical model above:
+
+  * The corpus is a collection of :math:`D` documents.
+  * A document is a sequence of :math:`N` words.
+  * There are :math:`K` topics in the corpus. 
+  * The boxes represent repeated sampling. 
+
+
+The three levels of the Bayesian model are: 
+
+  1. Generation of global hyperparameters :math:`\alpha` and :math:`\eta`, 
+     and global parameter :math:`\beta`.
+
+  2. Generation of document-level variable :math:`\theta`. 
+
+  3. Generation of word-level variables :math:`z` and :math:`w`.
+
+
+In the graphical model, each node is a random variable and has a role in the 
+generative process. A shaded node indicates an observed variable and an unshaded 
+node indicates a hidden (latent) variable. In this case, words in the corpus are 
+the only data that we observe. The latent variables determine the random mixture 
+of topics in the corpus and the distribution of words in the documents. 
+The goal of LDA is to use the observed words to infer the hidden topic 
+structure. 
+
+
 When modeling text corpora, the model assumes the following generative process for
-a corpus with :math:`D` documents and :math:`K` topics:
+a corpus with :math:`D` documents and :math:`K` topics, with :math:`K` corresponding 
+to :attr:`n_components` in the API:
 
-  1. For each topic :math:`k`, draw :math:`\beta_k \sim \mathrm{Dirichlet}(\eta),\: k =1...K`
+  1. For each topic :math:`k \in K`, draw :math:`\beta_k \sim \mathrm{Dirichlet}(\eta)`. 
+     This provides a distribution over the words, i.e. the probability of a word appearing 
+     in topic :math:`k`. :math:`\eta` corresponds to :attr:`topic_word_prior`. 
 
-  2. For each document :math:`d`, draw :math:`\theta_d \sim \mathrm{Dirichlet}(\alpha), \: d=1...D`
+  2. For each document :math:`d \in D`, draw the topic proportions :math:`\theta_d \sim \mathrm{Dirichlet}(\alpha)`. :math:`\alpha` corresponds to :attr:`doc_topic_prior`. 
```

Comment:
please wrap to keep lines under 80 chars

Reply:
done